src/FunctionApps/python/requirements.txt

```diff
@@ -4,7 +4,7 @@ azure-functions
 azure-identity
 azure-storage-blob
 hl7
-phdi_building_blocks @ git+https://github.com/CDCgov/prime-public-health-data-infrastructure@main#subdirectory=src/lib/phdi-building-blocks
+phdi_building_blocks @ git+https://github.com/CDCgov/phdi-sdk#subdirectory=/phdi
```

Comment:
You're getting an error for this, but I think it should work if you remove the subdirectory... /phdi contains the code, but the project root (with pyproject.toml) is at the root level.